devdocs/godmode.md

```diff
@@ -0,0 +1,77 @@
+#GodMode User Notes
+
+##What is GodMode?
+
+Typically browser tests involve comparing the source of the webpage to an existing *expected* source. This serves to verify the DOM structure (HTML) of the page. However, creating new browser tests and updating outdated ones require considerable effort on the developer's part. In addition, ensuring that the generated *expected* source code works across computers, browsers, user accounts, and execution times is harder still. To overcome this problem, GodMode provides a simple way to create and update *expected* source files for browser tests and ensure the necessary cross-compatibility.
```

Comment:
GodMode standardizes the spacing, substitutes user accounts and dynamic timestamps with placeholders. This allows for the generated htmls to be cross-compatible